owncloudApp/build.gradle

```diff
@@ -76,6 +76,9 @@ dependencies {
     // Room
     implementation "androidx.room:room-runtime:$roomVersion"
     kapt "androidx.room:room-compiler:$roomVersion"
+
+    // Kotlin dependency injector
+    implementation 'org.koin:koin-androidx-viewmodel:2.0.0'
```

Comment:
👍

Reply:
There is a new version ;-)

Reply:
Updated

Reply:
BTW, Why don't you create a koin_version constant and use it here?

Reply:
Done